main_comment.go

```diff
@@ -188,10 +191,12 @@ func parsePrOptions(commentBody string) (map[string]string, error) {
 							" somerepo/pull/12/head --pr somerepo/1.0.x ",
 					)
 				}
-				prRepos[userInputParts[0]] = revision
+				buildOptions.PullRequests[userInputParts[0]] = revision
 			}
+		} else if word == "--fast" {
+			buildOptions.Fast = true
```

Comment:
general comment: it needs an entry in the online help: https://github.com/mendersoftware/integration-test-runner/blob/master/main_pullrequest.go#L135

Reply:
https://github.com/mendersoftware/integration-test-runner/pull/175/commits/71bf5f3576d59083a7633666d275a1ab68f6168d